packages/html/src/pager/pager.spec.tsx

```diff
@@ -108,6 +108,7 @@ export const Pager = (
                             'k-pager-numbers'
                         )}>
                         <Button
+                            selected
```

Comment:
Any particular reason to leave the `selected` state ? In genera, if we remove the states from the specs, we should not remove them selectively.

Reply:
Pager component has one `disabled` and one `selected` button by default in all suites, so if we remove these states, we will have to add exceptions in the test, so it seamed reasonable to leave them. Though, I am okay to remove them as well.